### ISSUE TITLE
Directing output to new chat windows?

### ISSUE THREAD

**Issue by eartsar** [joemcbride/outlander-osx#54]
Perhaps this is in the documentation, but how is one meant to filter text to output to a particular window? For example, inventory output, or combat assess output?

**Comment by joemcbride** [comment]
I don't think there is a way with how the windows work currently, for those particular items.  Inventory would be pretty easy to add, I'll take a look at the combat assesses.

**Comment by joemcbride** [comment]
Just out of curiosity btw, what version of OSX are you using?  I'm trying to get a feel of what versions people are using since that determines some of the features I can use.

**Comment by joemcbride** [comment]
Looks like assess will work right now.  Inventory doesn't look like it will work without some changes from me.  Will look at it more though.

Run the following command and it will create a new window for assesses.
```
#window add assess
```

**Comment by joemcbride** [comment]
Yeah confirmed that getting inventory to work will take some code changes.

**Comment by joemcbride** [comment]
One other option you have is to use the `#echo` command to direct output to a specific window.  You can use it coupled with the `action` command in scripts or global triggers.

The following global trigger will execute when the text `testing` is parsed.  You can test the trigger by typing `#parse testing` in the command prompt.  Triggers accept regex just like actions.  This trigger should print out `yap` colored in yellow.  You can also see I have a trigger for running a script on first login (for things such as sorting inventory).

![screen shot 2015-10-14 at 12 47 54 pm](https://cloud.githubusercontent.com/assets/255007/10495659/cd3c24ac-7271-11e5-80e8-0d988911ff20.png)
![screen shot 2015-10-14 at 12 50 41 pm](https://cloud.githubusercontent.com/assets/255007/10495736/35404d62-7272-11e5-992c-8b839adc2956.png)


* `#echo [>targetWindow] [color[,backgroundcolor]] <text>`
    * echo a message to a window
    * text - The text you want to echo
    * window (optional) - The target window you want to echo to.  Defaults to main.
    * color (optional) - The foreground color.  Use hex color.
    * backgroundcolor (optional) - The background color.
```
put #echo >thoughts #000000,#efefef hello
```

**Comment by eartsar** [comment]
Running the latest version of OSX (10.11).

Good to hear about more out-of-the-box windows. Assess would definitely be one of the more useful ones, I think.

**Comment by joemcbride** [comment]
Good news, this is resolved in version 0.9.0!

https://github.com/joemcbride/outlander-osx/releases/tag/v0.9.0

**Comment by eartsar** [comment]
you're amazing
